scopes/lanes/ui/readme/lane-readme.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
I would call this lanes-overview or something. it doesnt have anything to do with the readme. it just renders the readme or the gallery. probably related to why you chose to rename it in the index?

anyway, we should separate the readme from the overview